protocol/message.go

```diff
@@ -124,15 +137,23 @@ type DirectoryProofs struct {
 	STR []*DirSTR
 }
 
+// An STRList response includes a list of signed tree roots
+// STR. A CONIKS auditor returns this DirectoryResponse type upon an
+// AudutingRequest.
+type STRList struct {
```

Comment:
I think it would be better if we have something match with current `DirectoryProof(s)`. The reason we chose this name was to show the purpose of the response (I'm not sure 100% about this). Here are some come to my mind: `STRChain` or `ConsistencyProof`.